### PR TITLE
Plural getter/add pairs

### PR DESCRIPTION
**Pull request by avgeeklucky**
Add support for getter / add pairs named following English grammar rules (e.g. plural getter - `getChildren`, singular setter `addChild`). [doctrine/inflector](https://github.com/doctrine/inflector/blob/2.0.x/docs/en/index.rst) is used to singularize plural getters find their corresponding add method.

Example:
```php
class Category {
    ...
    /**
     * @return Product[]
     */
    public function getProducts(): array
    {
        return $this->products;
    }

    public function addProduct(Product $product): void
    {
        $this->products[] = $product;
    }
    ...
}
```

### CODE REVIEW

**Review by PrinsFrank — APPROVED** [review]

**Comment by PrinsFrank** [comment]
Awesome! @bram123 can you also take a look?

**Comment by bram123** [comment]
@avgeeklucky 
Can you create an extra UT on how it will behave with multiple accessorpairs in one class?
So the existing GetSetAdd data file, but with setProperties instead.

File: tests/Unit/Constraint/MethodPair/AccessorPair/AccessorPairProviderTest.php [hunk 1]
```diff
@@ -31,13 +31,13 @@ public function testGetAccessorPairs(DataInterface $class): void
         $actualPairs = $provider->getAccessorPairs(new ReflectionClass($class));
 
         $expectedPairs = $class->getExpectedPairs();
-        static::assertCount(count($expectedPairs), $actualPairs);
+        static::assertCount(count($expectedPairs), $actualPairs, get_class($class));
```
Comment (bram123) [inline tests/Unit/Constraint/MethodPair/AccessorPair/AccessorPairProviderTest.php]: This should probably be an actual message instead of just the classname.
Why did you need this? The classname should already be used as a testcase name by the dataprovider.

**Comment by avgeeklucky** [comment]
@bram123 Added, works as expected.

**Review by bram123 — APPROVED** [review]
